### PR TITLE
Update analytics docs with new trigger structure

### PR DESCRIPTION
**Pull request by rudygalfi**
And some misc. cleanup

### CODE REVIEW

**Comment by rudygalfi** [comment]
@avimehta to review

cc @cramforce

File: extensions/amp-analytics/amp-analytics.md [hunk 1]
```diff
@@ -14,11 +14,11 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-# <a name="amp-analytics"></a>`amp-analytics`
+### <a name="amp-analytics"></a>`amp-analytics`
```
Comment (avimehta) [inline extensions/amp-analytics/amp-analytics.md]: Any reason to change the heading level?

File: extensions/amp-analytics/amp-analytics.md [hunk 2]
```diff
@@ -137,56 +138,85 @@ The `vars` attribute in the configuration can be used to define new key-value pa
...
 The `triggers` attribute describes when an analytics request should be sent. It contains a key-value pair of trigger-name and
  trigger-configuration. Trigger name can be any string comprised of alphanumeric characters (a-zA-Z0-9). Triggers from a
  configuration with lower precedence are overridden by triggers with the same names from a configuration with higher precedence.
 
-  - `on` (required) The event to listener for. Valid values are `visible`,`click`, and `timer`.
+  - `on` (required) The event to listener for. Valid values are `click`, `scroll`, `timer`, and `visible`.
   - `request` (required) Name of the request to send (as specified in the `requests` section).
   - `vars` An object containing key-value pairs used to override `vars` defined in the top level config, or to specify
     vars unique to this trigger.
+  - `selector` (required when `on` is set to `click`) This configuration is used on conjunction with the `click` trigger. Please see below for details.
```
Comment (avimehta) [inline extensions/amp-analytics/amp-analytics.md]: This works for now. But in future, visible might support selectors as an optional thing. Not sure how you would put that in this format.

**Comment by avimehta** [comment]
lgtm with a few comments.

cc @Meggin

File: extensions/amp-analytics/amp-analytics.md (hunk 2 above)
Comment (rudygalfi) [inline extensions/amp-analytics/amp-analytics.md]: At that point, we can rework it. The main point here is to give visibility into all of the possible children of `triggers` and the situations in which they are required/optional/irrelevant.

File: extensions/amp-analytics/amp-analytics.md (hunk 1 above)
Comment (rudygalfi) [inline extensions/amp-analytics/amp-analytics.md]: I changed it to be consistent with the rest of the extension .md files. They all currently start at level 3 and the main sections are level 4. I could see changing this back in the future, but for now wanted to get it consistent with the rest of the files.

**Pull request merged**